ivy/functional/ivy/meta.py

```diff
@@ -617,8 +620,10 @@ def reptile_step(
     ...     "latent": _variable(ivy.array([1.0, 2.0]))
     ... })
 
-    >>> cost, gradients, firsts = ivy.reptile_step(batch, inner_cost_fn, variables, 4, 0.025,
-    ...                                            batched=False, num_tasks=2, return_inner_v='first')
+    >>> cost, gradients, firsts = ivy.reptile_step(batch,
+                                                   inner_cost_fn, variables, 4, 0.025,
+    ...                                            batched=False,
+                                                   num_tasks=2, return_inner_v='first')
```

Comment:
Note that this is an invalid documentation. Python examples should start with `>>>` or `...`

Reply:
thanks